### PR TITLE
MODLD-333: Send for indexing all the Instances related to Work

### PR DESCRIPTION
**Pull request by askhat-abishev**

### CODE REVIEW

File: src/main/java/org/folio/linked/data/integration/ResourceModificationEventListener.java [hunk 1]
```diff
@@ -28,7 +28,7 @@ public class ResourceModificationEventListener {
   @TransactionalEventListener
   public void afterCreate(ResourceCreatedEvent resourceCreatedEvent) {
     log.info("ResourceCreatedEvent received [{}]", resourceCreatedEvent);
-    kafkaSender.sendSingleResourceCreated(resourceCreatedEvent.work());
+    kafkaSender.sendSingleResourceCreated(resourceRepository.getReferenceById(resourceCreatedEvent.work().getId()));
```
Comment (pkjacob) [inline src/main/java/org/folio/linked/data/integration/ResourceModificationEventListener.java:31]: We should do the same for `afterUpdate` also right?

**Review by PBobylev — APPROVED** [review]

File: src/main/java/org/folio/linked/data/integration/ResourceModificationEventListener.java (hunk 1 above)
Comment (askhat-abishev) [reply src/main/java/org/folio/linked/data/integration/ResourceModificationEventListener.java:31]: @pkjacob I don't think so

**Review by pkjacob — APPROVED** [review]

**Review by aleksei-pronichev — CHANGES_REQUESTED** [review]

File: src/main/java/org/folio/linked/data/integration/ResourceModificationEventListener.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (askhat-abishev) [reply src/main/java/org/folio/linked/data/integration/ResourceModificationEventListener.java]: done

File: src/test/resources/application-test-search.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+spring.kafka.producer.properties.retries: 0
```
[the remark this answers is not in the document]
Comment (askhat-abishev) [reply src/test/resources/application-test-search.yml]: done

**Review by aleksei-pronichev — APPROVED** [review]

**Pull request merged**